web/styles/message_header.css

```diff
@@ -16,7 +16,7 @@
         display: flex;
         align-items: center;
         justify-content: space-between;
-        height: 28px;
+        height: 2em; /* 28px at 14px em */
```

Comment:
Came across this while working on #32967. From what I can see in the inspector, the font-size on the message-header is set to 16px. So, here, the height actually equates to `32px` instead of `28px` as expected according to the accompanying comment.

Reply:
@evykassirer do you have this on your list?

Reply:
Sort of, in that I followed up on it, and am working on adjacent things right now, but it might have slipped through the cracks still so thanks for following up!

Reply:
@sayamsamal given the `15/14 em` font size above, I think the font-size here is actually set to 15px not 16px (at 14px em at least). Does that sound right?

Reply:
Confirmed: https://chat.zulip.org/#narrow/channel/6-frontend/topic/info.20density.20with.20em.20font.20size

#33192